src/requests/consumption.js

```diff
@@ -517,8 +523,10 @@ const createSolutions = async (resourceDetails, programDetails) => {
 
 		if (solutionCertificateMap && solutionCertificateMap.length > 0) {
 			solutionCertificateMap.forEach((solutionMap) => {
-				const found = createdSolutions.find((solution) => solution.externalId == solutionMap.externalId)
-				insertCertificateTemplate(found.certificate, found._id, programDetails._id)
+				const found = createdSolutions.find(
```

Comment:
@adithyadinesh0412  rename to foundSolution
ifsolution exist then only do insertCertificateTemplate